myst_nbs/case_studies/multilevel_modeling.myst.md

```diff
@@ -13,8 +13,8 @@ kernelspec:
 
 # A Primer on Bayesian Methods for Multilevel Modeling
```

Comment:
The myst target before the title is missing: https://docs.pymc.io/en/latest/contributing/jupyter_style.html#first-cell